content/06.methods.md

```diff
@@ -70,75 +69,72 @@ Tissues were lysed using a Qiagen TissueLyser II (Qiagen) with 2×30 sec at 18Hz
+We performed germline haplotype calls using the [GATK Joint Genotyping Workflow](https://software.broadinstitute.org/gatk/best-practices/workflow?id=11145) on individual GVCFs from the normal sample alignment workflow.
 Using only SNPs, we applied the [GATK generic hard filter suggestions](https://gatkforums.broadinstitute.org/gatk/discussion/2806/howto-apply-hard-filters-to-a-call-set) to the VCF, with an additional requirement of 10 reads minimum depth per SNP.
-This filtered VCF was used as input to Control-FREEC and CNVkit (below) for generation of BAF files.
+We used the filtered VCF as input to Control-FREEC and CNVkit (below) to generate B-allele frequency (BAF) files.
 GATK v4.0.12.0 was used for all steps except `VariantFiltration`, which used 3.8.0 because as of GATK 4.0.12.0, this tool was beta and known to be unreliable for this purpose.
-This single-sample workflow can be found in the [Kids First GitHub repository](https://github.com/kids-first/kf-jointgenotyping-workflow).
-References can be obtained from the [Broad Genome References on AWS](https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/) bucket, with a general description of references here [@url:https://s3.amazonaws.com/broad-references/broad-references-readme.html].
+This single-sample workflow is available in the [D3b GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows).
+References can be obtained from the [Broad Genome References on AWS](https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/) bucket, and a general description of references can be found at https://s3.amazonaws.com/broad-references/broad-references-readme.html.
 
 #### Somatic Mutation Calling
 
 ##### SNV and indel calling
 
-For PBTA samples, we used four variant callers to call SNVs and indels from targeted DNA panel, WXS, and WGS data: Strelka2 [@doi:10.1038/s41592-018-0051-x], Mutect2 [@doi:10.1101/861054], Lancet  [@doi:10.1038/s42003-018-0023-9], and VarDict [@doi:10.1093/nar/gkw227].
-WXS samples from TCGA were run using Strelka2, Mutect2 and Lancet.
-TCGA samples were captured using different WXS target capture kits and all the BED files were downloaded from [`GDC portal`](https://api.gdc.cancer.gov/files).
-The input interval BED files for both panel and WXS data for PBTA  samples were provided by the manufacturers.
-For both PBTA and TCGA, all panel and WXS  BED files were padded by 100 bp on each side during Strelka2, Mutect2, and VarDict runs and 400 bp for the Lancet run.  
+For PBTA samples, we used four variant callers to call SNVs and indels from panel, WXS, and WGS data: Strelka2 [@doi:10.1038/s41592-018-0051-x], Mutect2 [@doi:10.1101/861054], Lancet [@doi:10.1038/s42003-018-0023-9], and VarDict [@doi:10.1093/nar/gkw227].
+VarDict-only calls were not retained since ~ 39M calls with low VAF were uniquely called and may be potential false positives.
```

Comment:
```suggestion
We did not retain VarDict-only calls because ~ 39M calls with low VAF were uniquely called, suggesting they may have been false positives.
```